packages/core/lib/configureGanacheOptions.js

```diff
@@ -0,0 +1,48 @@
+function sanitizeGanacheOptions(ganacheOptions) {
+  const network_id = ganacheOptions.network_id;
+
+  // Use default network_id if "*" is defined in config
+  if (network_id === "*") {
+    return { ...ganacheOptions, network_id: Date.now() };
+  }
+
+  const parsedNetworkId = parseInt(network_id, 10);
+  if (isNaN(parsedNetworkId)) {
+    const error =
+      `The network id specified in the truffle config ` +
+      `(${network_id}) is not valid. Please properly configure the network id as an integer value.`;
+    throw new Error(error);
+  }
+  return { ...ganacheOptions, network_id: parsedNetworkId };
+}
+
+function configureManagedGanache(config, networkConfig, mnemonic) {
+  const ganacheOptions = {
```

Comment:
I notice `gasLimit` is missing... that needs to be added.  We shouldn't accept `gas` as a synonym for it, but we should accept `gasLimit`.

Reply:
Sure, I will add that. May be I misunderstood that it should be removed like it was previously done in `truffle test`.